### PR TITLE
Update the "about" with new "communities" section

### PR DESCRIPTION
**Pull request by imbaah**
@dakotabenjamin 

Fingers crossed

### CODE REVIEW

File: app/_data/es.yml [hunk 1]
```diff
@@ -322,6 +322,11 @@ about:
     text1:      "La adhesión al proyecto está abierta a cualquier ONG, institución de educación o grupo de la sociedad civil dispuestos a contribuir a los objetivos y cumplir con la ética expuesta anteriormente. La aprobación de la adhesión es responsabilidad de las organizaciones actualmente afiliadas."
     text2:      "Si estás interesado en adherirse como miembro, por favor ponte en contacto con nosotros mediante el siguiente formulario."
   contact_btn:  "CONTÁCTANOS PARA CONVERTIRTE EN MIEMBRO"
+   communities:
```
Comment (dakotabenjamin) [inline app/_data/es.yml:325]: Tiny issue here, the indentation is incorrect. Should be at the same level as the previous line

File: app/_includes/about.html [hunk 2]
```diff
@@ -82,6 +82,20 @@ <h1 class="title feature-header">{{site.data[locale].about.title}}</h1>
...
+				<div class="about-textbox">
+					<p style="text-align: left">{{site.data[locale].about.communities.text1}}</p>
+					<p style="text-align: left">{{site.data[locale].about.communities.text2}}</p>
+				</div>
+			</div>
+
+			<div class="about-contact center-text">
+				<a href="https://drive.google.com/drive/folders/1U80TG2elfc9JNnLoKVmbU1mDcK4zCaw6?usp=share_link" class="btn btn-blue">{{site.data[locale].about.reach_btn}}</a>
+			</div>
+
```
Comment (dakotabenjamin) [inline app/_includes/about.html:98]: this all looks great in my local test- only issue is the formatting may not be ideal. To fix, you would want to move the `</div>` on line 79 to line 98.

Comment (dakotabenjamin) [reply app/_includes/about.html:98]: current: 
image


with above fix:
image

**Review by dakotabenjamin — CHANGES_REQUESTED** [review]
One minor change needed and one formatting suggestion. Otherwise looks great!

File: app/_includes/about.html (hunk 2 above)
Comment (imbaah) [reply app/_includes/about.html]: Thank you! Followed your instructions. Hope all is good now.

**Pull request merged**